sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -140,6 +142,30 @@ def _get_unit_price(sku: Dict[str, Any]) -> float:
     return units + nanos
 
 
+def filter_zones(func: Callable[[], List[str]]) -> Callable[[], List[str]]:
+    """Decorator to filter the zones returned by the decorated function.
+    It first intersects the result with the global ZONES (if defined) and then
+    removes any zones present in the global EXCLUDED_REGIONS (if defined).
+    """
+
+    def wrapper(*arguments, **kwargs) -> List[str]:  # Renamed args to arguments
```

Comment:
Let's go with `args` and disable the linting error with `pylint: disable=redefined-outer-name`. I think that it's better to stick with the established `args` convention for this decorator, and it's pretty clear that it does not interact with the cli `args`.

Reply:
done